libraries/ag-charts-test/src/canvas/mock-canvas.ts

```diff
@@ -23,6 +27,7 @@ export class MockContext {
         this.realCreateElement = realCreateElement;
         this.ctx = {
             nodeCanvas,
+            getRenderContext2D: () => this.ctx.nodeCanvas.getContext('2d') as unknown as CanvasRenderingContext2D,
```

Comment:
Note: The TS compiler is reporting that the real and mock canvases are not compatible (which isn't what I would expect from a full mock). This isn't an ideal solution, but it's better that adding `as` in every `.test.ts` file that uses the `CanvasRenderingContext2D` object.